src/shared/styles/AboutStyles.js

```diff
@@ -0,0 +1,177 @@
+        },
+      
+        [theme.breakpoints.down('xs')]: {
+          width: '100%',
+          height: 'auto',
+        },
+        '@media (max-width:935px)': {
+          width: '90%',
+          margin: '18px',
+          height: 'auto',
+        },
+        '@media (max-width:650px)': {
+          width: '100%',
+          margin: 18,
+          height: 'auto',
+        },
+
+      },
+      
+      avatar: {
+        width: '60%',
+        height: 100,
+        marginLeft: '30px'
+      },
+
+      paragraph: {
+        color: '#726C6C', padding: '30px 0',
+        [theme.breakpoints.down('xs')]: {
+          width: '100%'
+        },
+        '@media (max-width:935px)': {
+          width: '90%',
+          margin: '18px',
+        },
+        '@media (max-width:650px)': {
+          width: '100%',
+          margin: 18,
+        },
+      },
+
+      righside: {
+        display: 'flex',
+        [theme.breakpoints.down('xs')]: {
+          width: '100%'
+        },
+        '@media (max-width:935px)': {
+          width: '100%',
+          margin: '18px',
+        },
+        '@media (max-width:650px)': {
+          width: '100%',
+          
+        },
+      },
+
+      cardContant: {
+        alignItems: 'center',
+        alignContent: 'center'
+      },
+      dotContianer: {
+        marginLeft: '320px',
+         marginTop: '30px'
+      },
+      bookButton: {
+        background: '#2196F3',
+        fontFamily: 'Poppins, sans-serif',
+        height: '35px',
+        width: '200px',
+        marginLeft: '250px',
+        marginTop: '50px',
+        borderRadius: '20px',
+        marginBottom: '50px',
+
+        [theme.breakpoints.down('xs')]: {
+         
+          marginLeft: '100px',
+        },
+        '@media (max-width:935px)': {
+         
+          marginLeft: '100px',
+        
+        },
+        '@media (max-width:650px)': {
+          marginLeft: '100px',
+          
+        },
+      },
+
+      hotelContainer: {
+        marginTop: '148px',
+        position: 'absolute',
+        marginLeft: '100px',
+        [theme.breakpoints.down('xs')]: {
+          width: '100%',
+          margin: 0
+        },
+        '@media (max-width:935px)': {
+          width: '100%',
+          
+          margin: 0
+        },
+        '@media (max-width:650px)': {
+          width: '100%',
+          margin: 0
+          
+        },
+      },
+
+      hotelImage: {
+        width: 400, height: 250,
+        [theme.breakpoints.down('xs')]: {
+          width: '93%'
+        },
+        '@media (max-width:935px)': {
+          width: '93%',
+          margin: '5px',
+        },
+        '@media (max-width:650px)': {
+          width: '93%',
+          margin: '5px'
+        },
+         
+      },
+      patternStyle: {
+        marginLeft: '30px',
+        position: 'absolute',
+        height: 200,
+        [theme.breakpoints.down('xs')]: {
+          width: '100%',
+          position: 'relative',
+        },
```

Comment:
<img width="455" alt="Screen Shot 2020-12-17 at 19 41 00" src="https://user-images.githubusercontent.com/54619678/102575330-970b4800-40fb-11eb-85e5-ebc97c433e10.png">

The width of the content is becoming larger than that of the body on small size devices. Could you mind `display none` that pattern   down **`xs`** since it's not only making the width greater but also somehow useless and annoying since you're displaying it behind the image.